include/library/spdm_common_lib.h

```diff
@@ -615,16 +615,16 @@ void libspdm_register_device_buffer_func(
  * The APP message format is defined by the transport layer.
  * Take MCTP as example: APP message == MCTP header (MCTP_MESSAGE_TYPE_SPDM) + SPDM message
  *
- * @param  spdm_context  A pointer to the SPDM context.
- * @param  session_id    Indicates if it is a secured message protected via SPDM session.
- *                       If session_id is NULL, it is a normal message.
- *                       If session_id is not NULL, it is a secured message.
- * @param  is_app_message  Indicates if it is an APP message or SPDM message.
- * @param  is_requester    Indicates if it is a requester message.
- * @param  message_size    Size in bytes of the message data buffer.
- * @param  message         A pointer to a source buffer to store the message.
- *                         For normal message, it shall point to the acquired sender buffer.
- *                         For secured message, it shall point to the scratch buffer in spdm_context.
+ * @param  spdm_context            A pointer to the SPDM context.
+ * @param  session_id              Indicates if it is a secured message protected via SPDM session.
+ *                                 If session_id is NULL, it is a normal message.
+ *                                 If session_id is not NULL, it is a secured message.
+ * @param  is_app_message          Indicates if it is an APP message or SPDM message.
+ * @param  is_requester_message    Indicates if it is a requester message.
```

Comment:
I would say
`is_requester_message` -> `is_request_message`
`Indicates if it is a requester message.` -> `Indicates if it is a request message.`
but others might have a different opinion.

Reply:
done